components/tooltip.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { css } from "react-emotion";
+import { globalTheme } from "../theme";
+
+const tooltipTextStyle = css`
+  font-size: 14px;
+  font-weight: normal;
+  text-align: left;
+  visibility: hidden;
+  width: 160px;
```

Comment:
We might want the tooltip text width to be configurable. Fine to wait and see when we use it for the health question.